src/ray/object_manager/plasma/store_runner.cc

```diff
@@ -133,6 +133,20 @@ bool PlasmaStoreRunner::IsPlasmaObjectSpillable(const ObjectID &object_id) {
 
 int64_t PlasmaStoreRunner::GetConsumedBytes() { return store_->GetConsumedBytes(); }
 
+void PlasmaStoreRunner::EvictObjectsIfPossible() {
+  RAY_LOG(DEBUG) << "Attempting to evict objects from the plasma store";
+  if (evicting_) {
+    return;
+  }
+  evicting_ = true;
+  main_service_.post(
+      [this]() {
+        store_->EvictObjectsIfPossible();
+        evicting_ = false;
```

Comment:
should evicting_  be guarded as it's been accessed in different thread?

Reply:
Looks like it. Unfortunate that lock annotations can't help with these kinds of threading bugs.

Reply:
Changed to atomic bool.